scripts/ci/docker-compose/base.yml

```diff
@@ -34,6 +34,8 @@ services:
     ports:
       - "${WEBSERVER_HOST_PORT}:8080"
       - "${FLOWER_HOST_PORT}:5555"
+    cap_add:
```

Comment:
`SYS_PTRACE` isn't necessary to fix the issue. I added this because it makes it possible to run `py-spy dump` in breeze. @potiuk let me know if you have concerns. I can remove this change.

Reply:
Looks good no problem with capability added :)